cw/cw305/simple_capture_traces.py

```diff
@@ -34,6 +35,19 @@ def initialize_capture(device_cfg, spiflash_cfg):
     device_cfg['pll_frequency'],
     device_cfg['baudrate'])
   print(f'Scope setup with sampling rate {ot.scope.clock.adc_rate} S/s')
+  # Ping target
+  print("Reading from FPGA using simpleserial protocol.")
+  version = None
+  ping_cnt = 0
+  while not version:
+    if ping_cnt == 3:
+      print(f"Error: No response from the target (attempts: {ping_cnt}).", file=sys.stderr)
+      sys.exit(1)
```

Comment:
Should this throw an error instead, in case this is used in the context of a library?

Reply:
Good point, done.